README.md

```diff
@@ -63,17 +63,22 @@ Make sure this file is stored at the root of the project.
        >Note: This command will automatically assign RBAC contributor role to the service principal at subscription level.
        You can narrow down the scope to the specific resource group where your tests will create the resources.
 
-    1. Copy the output contents and paste it in a file called azureauth.json, and secure it with file system permissions.
-    1. Set an environment variable pointing to the file path you just created. Here is an example with Powershell and bash:
-        
-        Powershell
-        ```powershell
-        [Environment]::SetEnvironmentVariable("AZURE_AUTH_LOCATION", "C:\sdksample\azureauth.json", "User")
-        ```
-        Bash
-        ```bash
-        export AZURE_AUTH_LOCATION=/sdksamples/azureauth.json
-        ```
+   1. Set the following environment variables from the output of the creation:
+
+      Powershell
+       ```powershell
+       $env:AZURE_SUBSCRIPTION_ID = <subscriptionId>
+       $env:AZURE_CLIENT_ID = <clientId>
+       $env:AZURE_CLIENT_SECRET = <clientSecret>
+       $env:AZURE_TENANT_ID = <tenantId>
+       ```
+      Bash
+       ```bash
+       export AZURE_SUBSCRIPTION_ID=<subscriptionId>
+       export AZURE_CLIENT_ID=<clientId>
+       export AZURE_CLIENT_SECRET=<clientSecret>
+       export AZURE_TENANT_ID=<tenantId>
```

Comment:
At least one sample must use the file authentication so customers can continue to see how it is done, let's leave this one using the way it was before this update.

Reply:
It looks like the newest versions of the SDK do not support ServiceClientCredentials.
I was not able to authenticate with it and therefor ended up authenticating as the SDK documentation recommend.